Manual/HTML/Logical Color Combinations.html

```diff
@@ -0,0 +1,450 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>Logical Color Combinations — Keep Talking and Nobody Explodes Module</title>
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="Css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+    <script src="js/jquery.3.7.0.min.js"></script>
+    <script src="js/Utilities/svg-utils.js"></script>
+    <style>
+        .dark table, table {
+            margin: 1em auto;
+            border: none;
+        }
+
+        .led-table { text-align: center; }
+        .operator-table { text-align: center; }
+        .operator-table td { font-size: 80%; }
+    </style>
+</head>
+<body>
+    <div class="section">
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">The Cruel Modkit</span>
+            </div>
+            <div class="page-content">
+                <h2>On the Subject of Logical Color Combinations</h2>
+                <p class="flavour-text">Who knew logic gates become so much harder when you are colorblind?</p>
+                <p><strong>Note: </strong>If your module’s components aren’t exactly <strong>[LEDs, Arrows, Button]</strong>, you are looking at the wrong section of this manual!</p>
+                <ul>
+                    <li>To disarm the module, submit the correct combinations of LED colors and arrow directions to submit the correct Base 36 number.</li>
+                    <li>To begin the module, press ❖. A sequence of 8 colors will flash on the arrow module.</li>
+                    <li>Pressing ❖ again will repeat the same sequence.</li>
+                    <li>Pair each flash with the corresponding LED from left to right.</li>
+                    <li>Find each pair in the table below where the LED color is the row, the flashing color is the column, and the color of the module’s arrow’s center button is the main column.</li>
+                    <li>The intersection is a Base 36 digit. (Zeros are marked with a *)</li>
+                </ul>
+                <table class="led-table">
+                    <tr>
+                        <th class="corner" rowspan="2"></th>
+                        <th colspan="4">White</th>
+                        <th colspan="4">Grey</th>
+                    </tr>
+                    <tr>
+                        <th>RED</th>
+                        <th>YELLOW</th>
+                        <th>BLUE</th>
+                        <th>GREEN</th>
+                        <th>RED</th>
+                        <th>YELLOW</th>
+                        <th>BLUE</th>
+                        <th>GREEN</th>
```

Comment:
These cells could use the actual color as the background color so it looks a bit nicer (like in Metered Button and Who's Who's manuals). If you do add this, make sure that you use the same classes used in the other manuals but add another one for grey